Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Loading/LoadingPage.xaml.cs

```diff
@@ -62,9 +64,13 @@ private void Load()
                 if (loadingContentControl != null)
                 {
                     loadingContentControl.ContentTemplate = resources["LogoTemplate"] as DataTemplate;
-                    await loadingContentControl.Blur(2, 100).StartAsync();
+
+                    // We need to blur the ScrollViewer hosting the main content of the page and not the
+                    // root control, otherwise the popup with the icon would be blurred as well, as it
+                    // would be in the same visual tree as the control whose visual is being blurred.
+                    await scrollViewerControl.Blur(2, 100).StartAsync();
```

Comment:
I think the original intent here was to try and keep the same pattern as the other scenarios; however I'm not sure when this stopped working. 

I think it'd be easier to just have them all blur by setting the background property (L65 in the XAML) to a blur/pipeline brush instead of using the code-behind? 🙂 Thoughts?

Reply:
The issue with swapping the brush entirely is that the `PipelineBrush` doesn't support animations, so we wouldn't be able to have this same fade in/out effect we have here. I mean that'd be possible with a `PipelineBuilder`, but that'd result in even more code behind than what we already have here, as we'd have to both create the pipeline, store the brush, swap it, store the animation delegate to invoke, etc. I feel like for this specific scenario just using this approach is likely the easiest solution, and it just results in two lines of code anyway 🤔

Reply:
@Sergio0694 I'm saying to remove the animation of the blur entirely, the loading control itself already has a pop-up/opacity animation (or something), so if we just set the example like this (though maybe with a slightly simpler brush):

```xml
    <controls:Loading x:Name="LoadingControl">
      <controls:Loading.Background>
        <brushes:PipelineBrush Source="{effects:BackdropSource}">
          <effects:LuminanceToAlphaEffect/>
          <effects:OpacityEffect Value="0.4"/>
          <effects:BlendEffect Mode="Multiply" Source="{effects:BackdropSource}"/>
          <effects:BlurEffect Amount="16"/>
          <effects:ShadeEffect Color="#FF222222" Intensity="0.2"/>
          <effects:BlendEffect Mode="Overlay" Source="{effects:TileSource Uri=ms-appx:///Assets/BrushAssets/NoiseTexture.png}"/>
         </brushes:PipelineBrush>
      </controls:Loading.Background>
      <ContentControl x:Name="LoadingContentControl" />
    </controls:Loading>
```

We get the same effect without any code-behind at all (outside our template swapping and faking a time delay in the sample).

Reply:
Did some changes in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3488/commits/e02308b2f7ef5a399d6a90b7a297fc066dcaad6c switching to `AcrylicBrush` in all three samples, let me know what you think! 😄